bears/image/ImageCompressionBear.py

```diff
@@ -0,0 +1,87 @@
+import os
+import subprocess
+import shutil
+import optimage
+
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+from dependency_management.requirements.PipRequirement import PipRequirement
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+from coala_utils.FileUtils import create_tempfile
+
+
+class ImageCompressionBear(GlobalBear):
+    LANGUAGES = {'Image'}
+    REQUIREMENTS = {PipRequirement('optimage', '0.0.1'),
+                    DistributionRequirement(apt_get='jpegoptim'),
+                    DistributionRequirement(apt_get='libjpeg-progs',
+                                            binary='jpegtran'),
+                    DistributionRequirement(apt_get='pngcrush'),
+                    DistributionRequirement(apt_get='optipng')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'The coala developers'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Compression', 'Useless metadata'}
+
+    @classmethod
+    def check_prerequisites(cls):
```

Comment:
The reason for this is the fact `DistributionRequirement` doesn't implement `is_installed`. Related issue: https://gitlab.com/coala/package_manager/issues/11

Reply:
Fixed upstream